src/fmmax/fmm.py

```diff
@@ -720,6 +731,8 @@ def _numerical_eigensolve(
         transverse_permeability_matrix: The fourier-transformed transverse permeability
             matrix.
         expansion: The field expansion to be used.
+        tangent_vector_field: this argument merely exists to pass the tangent vector
```

Comment:
This should primarily describe what the arg is. Something like, "The tangent vector field used to generate the transverse permittivity and permeability matrices. This is an attribute of the returned `LayerSolveResult`.

Reply:
Got it, will make the change.